internal/types/client_utils.go

```diff
@@ -37,6 +37,74 @@ func ToStateAuthenticationPolicyContract(r *client.AuthenticationPolicyContract)
 	return caSliceOfObj, eaSliceOfObj
 }
 
+func AttrValueToBoolPointer(val attr.Value) *bool {
+	boolPointerVal := ConvertToPrimitive(val).(bool)
```

Comment:
Probably better to do this by converting to a types.Bool and then using the ValueBoolPointer method

Reply:
Removed this completely as it's not used.